tests/functional/clients/standalone/package_list/test_alpine.py

```diff
@@ -23,7 +22,7 @@ def test_pkgs_all(self, analyzed_data, pkg, version):
     def test_pkgs_allinfo(self, analyzed_data, pkg, metadata):
         result = analyzed_data("py38")
         pkgs = result['image']['imagedata']['analysis_report']['package_list']['pkgs.allinfo']['base']
-        loaded = dict(json.loads(pkgs.get(pkg, '{}')))
+        loaded = dict(pkgs.get(pkg, '{}'))
```

Comment:
I *think* this now needs to fallback to an empty dict, not a JSON object? 

```suggestion
        loaded = dict(pkgs.get(pkg, {}))
```

Reply:
yup yup, good catch